wolfcrypt/src/pkcs12.c

```diff
@@ -1161,7 +1267,100 @@ int wc_PKCS12_parse(WC_PKCS12* pkcs12, const char* psw,
                 ERROR_OUT(ASN_PARSE_E, exit_pk12par);
             }
 
-            /* decrypted content overwrites input buffer */
+
+#ifdef ASN_BER_TO_DER
+            /* If indefinite length format, ensure it is in the ASN format
```

Comment:
Pull this out into its own function in pkcs12.c.

Reply:
Done.